docker/shadow-hand/kinetic/intel_ide/Dockerfile

```diff
@@ -0,0 +1,13 @@
+#
+# Kinetic with build tools Dockerfile
+#
+# https://github.com/shadow-robot/sr-build-tools/
+#
+
+FROM shadowrobot/dexterous-hand:kinetic
+
+RUN \
+  sudo apt-get update && \
```

Comment:
sudo will be removed. Other docker files are using it so I thought I had to use it here as well